src/tools/pkcs11-tool.c

```diff
@@ -8635,6 +8632,22 @@ static void * test_threads_run(void * pttd)
 			}
 		}
 
+		else if (*pctest == 'L' && *(pctest + 1) == 'T') {
```

Comment:
oneline description of the LT flag would be good here in a comment. Also note about this new operation in the man page/help would be useful

Reply:
those options are currently not documented at all, the help message states to look at the source code... :roll_eyes:

Reply:
yeah. But in this case, it is not even documented in the source code, so please add a comment similarly as with the other options